src/apps/fee-list/list.tsx

```diff
@@ -29,6 +29,7 @@ const List: FC<ListProps> = ({
           {fees.map((itemData) => (
             <StackableFees
               amountOfMaterialsWithDueDate={itemData.materials.length}
+              item={{ faust: itemData.materials[0].recordId as FaustId }}
               faust={itemData.materials[0].recordId as FaustId}
```

Comment:
I did not updated fees.

They are FBS specific so still need a `faust` property. I have added `item` which is now needed for higher-order components to work.